src/olympia/devhub/templates/devhub/addons/listing/item_actions.html

```diff
@@ -47,7 +47,7 @@ <h5>{{ _('Actions') }}</h5>
       </li>
     {% endif %}
   {% endif %}
-  {% if addon.status != amo.STATUS_DISABLED and waffle.flag('beta-stats') %}
+  {% if addon.type in [amo.ADDON_EXTENSION, amo.ADDON_STATICTHEME] and addon.status != amo.STATUS_DISABLED and waffle.flag('beta-stats') %}
```

Comment:
can we define a constant for `[amo.ADDON_EXTENSION, amo.ADDON_STATICTHEME]` in amo and reference that instead?  I feel like we'll end up missing a check somewhere when it's expanded to other addon types and/or removed at a later point.

Reply:
Good idea.